Gopkg.toml

```diff
@@ -118,3 +118,7 @@
 [prune]
   go-tests = true
   unused-packages = true
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/GaryBoone/GoStats"
```

Comment:
This library was last updated 7 years ago: https://github.com/GaryBoone/GoStats

Do we really need this one and can we find something more maintained to replace this?

Reply:
this is statistics library, we can replace it, but I don't think it is critical

Reply:
Please replace it, we would like to use only well mantained libraries (active projects with many contributors and recommendations/stars)

Reply:
does this look like a good candidate https://github.com/montanaflynn/stats

Reply:
Yes, this looks better. I do not know if it is the best one - I let you do the research and choose by the criteria I mentioned.